src/views/Timeline.vue

```diff
@@ -205,63 +211,29 @@ export default {
 			}
 		},
 
-		/**
-		 * Return the props based on the element
-		 * Here we want to bind the full fileinfo
-		 * object so we stupidly return it whole!
-		 *
-		 * @param {Object} item the scoped item from the VirtualGrid
-		 * @returns {Object}
-		 */
-		getProps(item) {
-			return Object.assign({}, item, {
-				loadMore: this.fetchContent,
-			})
-		},
-
-		/**
-		 * Return the component based on the element
-		 * We only have files in the Timeline,
-		 * so we return Files!
-		 *
-		 * @returns {Object}
-		 */
-		getComponent() {
-			return File
-		},
-
-		debounceOnBottomReached: debounce(function() {
-			this.onBottomReached()
-		}, 1000),
-
-		/**
-		 * When virtual grid reach the bottom,
-		 * we load the next page
-		 */
-		onBottomReached() {
-			// if we're currently loading or if a previous
-			// request returned the last page, we stop
-			if (this.loadingPage || this.done) {
-				return
-			}
-
-			console.debug('Loading next page', this.page)
-			this.fetchContent()
-		},
-
 		/**
 		 * Reset this component data to a pristine state
 		 */
 		resetState() {
 			this.$store.dispatch('resetTimeline')
 			this.done = false
+			this.isEmpty = false
 			this.error = null
 			this.loadingPage = false
-			this.page = 0
-			this.page = 0
+			this.$refs.virtualgrid.resetGrid()
+		},
+
+		getFormatedDate(string, format) {
+			return moment(string).format(format)
 		},
 
 	},
 
 }
 </script>
+
+<style lang="scss" scoped>
+.grid-container {
+	padding: 66px;
```

Comment:
Padding is variable.
Please check https://github.com/nextcloud/photos/blob/b501e72f31bbaf90d89e3c55ef2438b002244d0b/src/assets/grid-sizes.js